src/queries.js

```diff
@@ -180,6 +180,22 @@ function queryByAltText(...args) {
   return firstResultOrNull(queryAllByAltText, ...args)
 }
 
+function queryAllByCurrentValue(
+  container,
+  value,
+  {exact = true, collapseWhitespace = true, trim = true} = {},
+) {
+  const matcher = exact ? matches : fuzzyMatches
+  const matchOpts = {collapseWhitespace, trim}
+  return Array.from(container.querySelectorAll(`input,textarea,select`)).filter(
+    node => matcher(node.value, node, value, matchOpts),
```

Comment:
This wont work for selects. Please look at (and maybe just use) the logic in `queryAllBySelectText` for selects.